web/pandas/pdeps/0001-purpose-and-guidelines.md

```diff
@@ -71,16 +82,62 @@ Proposing a PDEP is done by creating a PR adding a new file to `web/pdeps/`.
+`Status: Under discussion` by the author(s), when they are ready to proceed with the decision
+making process.
 
-#### Accepted PDEP
+#### PDEP Discussion Timeline
+
+A PDEP discussion will remain open for up to 60 days. This period aims to enable participation
+from volunteers, who might not always be available to respond quickly, as well as provide ample
+time to make changes based on suggestions and considerations offered by the participants.
+Similarly, the following voting period will remain open for 15 days.
+
+To enable and encourage discussions on PDEPs, we follow a notification schedule. At each of the
+following steps, the pandas team, and the pandas dev mailing list are notified via GitHub and
+E-mail:
+- Once a PDEP is ready for discussion.
+- After 30 discussion days, with 30 days remaining for discussion.
+- After 45 discussion days, with 15 days remaining for discussion.
+- In case 15 days passed without any new comments, the authors may close the discussion period
+  and open the voting period.
+- Once the voting period starts, after 60 days or in case of an earlier vote, with 15 days
+  remaining for voting.
+- After 10 voting days, with 5 days remaining for voting.
+
+#### Casting Votes
+
+As the voting period starts, a VOTE issue is created which links to the PDEP discussion pull request.
+Each voting member may cast a vote by adding one of the following comments:
 
-A PDEP can only be accepted by the core development team, if the proposal is considered
-worth implementing. Decisions will be made based on the process detailed in the
-[pandas governance document](https://github.com/pandas-dev/pandas-governance/blob/master/governance.md).
-In general, more than one approval will be needed before the PR is merged. And
-there should not be any `Request changes` review at the time of merging.
+- +1: approve.
+- 0: abstain.
+  - Reason: A one sentence reason is required.
+- -1: disapprove
+  - Reason: A one sentence reason is required.
+A disapprove vote requires prior participation in the PDEP discussion issue.
+
+Once the voting period ends, any voter may tally the votes in a comment, using the format: x-y-z,
+where x stands for the total of approving, y of abstaining, and z of disapproving votes cast.
+
+#### Quorum and Majority
+
+For a PDEP vote to result in accepting the proposal, a quorum is required. All votes (including
+abstentions) are counted towards the quorum. The quorum is computed as the lower of these two
+values:
+
+- 11 voting members.
+- 50% of voting members.
+
+Given a quorum, a majority of 75% of the non-abstaining votes is required as well, i.e. 75% of
+the approving and disapproving votes must be in favor.
+
+Thus, abstaining votes count towards a quorum, but not towards a majority. A voting member might
+choose to abstain when they have participated in the discussion, have some objections to the
+proposal, but do not wish to stop the proposal from moving forward, nor indicate their full
+support.
+
```

Comment:
Should there be an explicit statement discussing what happens if a quorum is not reached, i.e. if not enough voting members participate in the vote?
Does that mean the proposal is explicitly rejected and cannot be resubmitted?
Or does it mean that the PDEP may undergo a period of hiberation followed by a new period of discussion and a new vote is permitted after at least (4?) months of the previous vote?

Reply:
Why shouldn't it be possible to resubmit pdeps if circumstances change? How would decide if it's the same pdep or something different if some parameters are changed? I'd rather not worry about this now. Most PDEPs will be submitted by core-devs, and I think there isn't anyone in the group who wants to spam us with proposals

Reply:
> Should there be an explicit statement discussing what happens if a quorum is not reached, i.e. if not enough voting members participate in the vote?

We haven't discussed this possibility.

Reply:
We discussed this in our governance meeting today. We propose that not reaching quorum is equivalent to rejecting because effectively the PDEP was not accepted in time. The authors may decide to resubmit any rejected PDEP, as long as they add a link to the previous discussion in the new PDEP proposal. As before, we expect PDEP authors to use their own judgement when they decide to resubmit a PDEP proposal (or withdraw one), as to the ongoing relevance and likelihood of reaching quorum and/or acceptance for the resubmission. No hibernation period is explicitly required.
I'll add a change to reflect this soon.